perfscale_regression_ci/scripts/scalability/pod-affinity-anti-affinity.sh

```diff
@@ -0,0 +1,187 @@
+#/!/bin/bash
+################################################
+## Auth=prubenda@redhat.com lhorsley@redhat.com
+## Desription: This testcase tests Pod Affinity and anti-affinity as we approach pod capacity on compute nodes.
+## Polarion test case: OCP-18083 - NextGen Pod scheduler at capacity with Pod Affinity and Anti-Affinity rules	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-18083
+## Cluster config: Cluster needed in AWS EC2 (m5.xlarge, 3 master/etcd, 3 worker/compute nodes)
+## kube-burner config: perfscale_regerssion_ci/kubeburner-object-templates/pod-affinity-anti-affinity-config.yml
+################################################ 
+
+
+source ../../utils/run_workload.sh
+source ../custom_workload_env.sh
+source ../common.sh
+source pod-affinity-anti-affinity_env.sh
+
+
+export POD_ANTI_AFFINITY_JOB_ITERATION=${POD_ANTI_AFFINITY_JOB_ITERATION:-190}
+export POD_AFFINITY_JOB_ITERATION=${POD_AFFINITY_JOB_ITERATION:-190}
+
+function wait_for_running() {
```

Comment:
let's add this function to the scripts/common.sh file and parameterize it similar to wait_for_completion and wait_for_termination